src/client/cl_view.c

```diff
@@ -363,8 +363,8 @@ CL_PrepRefresh(void)
 
 	/* start the cd track */
 	int track = (int)strtol(cl.configstrings[CS_CDTRACK], (char **)NULL, 10);
-
-	OGG_PlayTrack(track);
+	
```

Comment:
Here are some superfluous whitespaces.

Reply:
I'll try to find what is causing this in my text editor.